src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```
@@ -404,6 +424,39 @@ def _get_linux_fx_version(cmd, resource_group_name, name, slot=None):
     return site_config.linux_fx_version
 
 
+def url_validator(url):
+    try:
+        result = urlparse(url)
+        return all([result.scheme, result.netloc, result.path])
+    except ValueError:
+        return False
+
+
+def _get_linux_multicontainer_decoded_config(cmd, resource_group_name, name, slot=None):
+    from base64 import b64decode
+    linux_fx_version = _get_linux_fx_version(cmd, resource_group_name, name, slot)
+    if not any([linux_fx_version.startswith(s) for s in MULTI_CONTAINER_TYPES]):
+        raise CLIError("Cannot decode config that is not one of the"
+                       " following types: {}".format(','.join(MULTI_CONTAINER_TYPES)))
+    try:
+        return b64decode(linux_fx_version.split('|')[1].encode('utf-8'))
+    except:
```

i suggest you don't catch, let it throw unless you can make the error specific